src/components/Timeline/TimelineIcon.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+import { makeStyles } from '@material-ui/core/styles'
+import TimelineItem from '@material-ui/lab/TimelineItem'
+import TimelineSeparator from '@material-ui/lab/TimelineSeparator'
+import TimelineConnector from '@material-ui/lab/TimelineConnector'
+import TimelineContent from '@material-ui/lab/TimelineContent'
+import TimelineDot from '@material-ui/lab/TimelineDot'
+
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import Avatar from 'cozy-ui/transpiled/react/Avatar'
+
+const useStyles = makeStyles(theme => ({
+  iconItem: {
+    marginTop: '-14px',
+    '&::before': {
+      padding: 0,
+      flexGrow: 0
+    }
+  },
+  iconDot: {
+    backgroundColor: theme.palette.background.paper,
```

Comment:
I really don't understand why sometimes we use the `theme` and sometimes we don't. https://github.com/cozy/coachCO2/pull/19/commits/3ecac1b1b78fb2f0180ce8635af9a565b61eb546#diff-e2fd9a9f7d3ebe18e82b3cf4a06f8c014edc9b0e47734075f747c2ad80825ca6R23

What is the rule?

Reply:
Y'a pas de règle si ce n'est qu'utiliser `theme` demande d'utiliser le hook `useTheme` ou `makeStyles` de mui. Quand ce n'est pas nécessaire, passer par les var revient au même